src/interchange/src/avro.rs

```diff
@@ -281,27 +324,85 @@ impl<'a> AvroDecode for DebeziumSourceDecoder<'a> {
                         DecodeError::Custom("\"lsn\" is not an integer".to_string())
                     })?);
                 }
+                // SQL Server
+                "change_lsn" => {
+                    let next = ValueDecoder;
+                    let val = field.decode_field(next)?;
+                    let val = match val {
+                        Value::Union { inner, .. } => *inner,
+                        val => val,
+                    };
+                    match val {
+                        Value::Null => {}
+                        Value::String(s) => {
+                            if let Some(i) = decode_change_lsn(&s) {
+                                change_lsn = Some(i);
+                            } else {
+                                return Err(AvroError::Decode(DecodeError::Custom(format!(
+                                    "Couldn't decode MS SQL LSN: {}",
+                                    s
+                                ))));
+                            }
```

Comment:
It's a bit weird to do that because I would have to make it return a `Result<MSSqlLsn, AvroError>`, and `decode_change_lsn` has nothing to do with Avro specifically.